client/src/partials/patients/record/patient_record.html

```diff
@@ -117,13 +117,78 @@
         <div class="col-sm-12">
           <div class="panel panel-default">
             <div class="panel-heading">
-              <span class="glyphicon glyphicon-file"></span> Latest Bill
+              <span class="glyphicon glyphicon-file"></span>
```

Comment:
Could you make this either a component, or a separate controller, like the other two panels?